tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/model/CardTable2.java

```diff
@@ -97,6 +100,7 @@ public record CardTable2(
 	MarkupString sc_lore,
 	MarkupString tc_lore,
 	MarkupString ko_lore,
+	MarkupString ygo_lore,
```

Comment:
Should be move bellow and be a simple String, as it is from a videogame that we don't consider here.